fineract-provider/src/main/java/org/apache/fineract/portfolio/client/service/ClientWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -195,8 +203,138 @@ private void handleDataIntegrityIssues(final JsonCommand command, final DataInte
     @Transactional
     @Override
     public CommandProcessingResult createClient(final JsonCommand command) {
-
-        try {
+    	 try {
+             final AppUser currentUser = this.context.authenticatedUser();
+             
+             final long configId=29;
```

Comment:
I am giving insert for this specific configuration and hence I have hardcoded it.